### ISSUE TITLE
feature: Copy and Pasting properties of object/layer

### ISSUE THREAD

**Issue by VikSin** [uxbox/uxbox#3754]
### Is your feature request related to a problem? Please describe.

Need option to copy properties of one object like border, fill, stroke, shadow & blur and pasting(applying) to another object. 

### Describe the solution you'd like.

Figma have options in edit menu or shortcut "Ctrl + Alt + c" for copying and "Ctrl + Alt + v" for pasting properties.
or 

For individual properties copying, selecting property input field will highlight and now copying and pasting to other object/layer

![image 5](https://github.com/penpot/penpot/assets/25245765/83004a29-5cb5-4b1b-a1dc-ba62a965bac4)

I prefer drag n drop properties field of one object/layer to another object for pasting properties -

https://github.com/penpot/penpot/assets/25245765/9bd665a6-8c47-41c8-b9a5-5f6cd0c085d2

**Comment by madalenapmelo-kp** [comment]
Hi @VikSin,

Thank you for the suggestion! This is being tracked on Taiga [here](https://tree.taiga.io/project/penpot/issue/3135).

Hopefully this is something we can consider in the near future!

**Comment by Ali1Jamal** [comment]
Yes, I'm designing a website and have just created three shadows. I need to apply these to other elements. It's hard to retype the three shadows for every element, so I'm searching for a copy and paste solution.
